advocacy_docs/partner_docs/VeritasGuide/02-PartnerInformation.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: 'Partner Information'
+description: 'Details for Veritas and NetBackup for PostgreSQL'
+
+---
+| &nbsp;     | &nbsp; |
+| ----------- | ----------- |
+| **Partner Name**      |  Veritas     |
+| **Partner Product**   |   NetBackup for PostgreSQL    |
+| **Web Site**      |   https://www.veritas.com/   |
+| **Version & Platform**   |   NetBackup for PostgreSQL 9.1: Linux, Windows     |
+| **Product Description**      |  Database backup/restore tool     |
```

Comment:
Add a more detailed description.  If you can't find one on website ask Jasmer.  He just got landing page information which should be in partner folder.  May have good description.

Reply:
Updated with description from partner site - landing page information not yet received from partner